### ISSUE TITLE
Provide common transportation mechanism for WebServer

### ISSUE THREAD

**Issue by brahmaroutu** [IBM/cap#14]
Currently only atom/xml is supported, we need something like JSONfeed or protobuf mechanisms(discussed by @cmluciano and me) to deliver content.

**Comment by cmluciano** [comment]
I mention protobuf so that we can remove some of the type conversions that will be common between difference language implementations. Unfortunately I have not found docs on the possibibility of accepting xml in protobuf natively.

**Comment by mikebrow** [comment]
Yes we need this, but we'll need to support both as the xml one is the standard.

**Comment by MHBauer** [comment]
I'd leave it as xml as long as possible. protobuf binds you to protobuf too much.
